app/web/src/newhotness/Onboarding.vue

```diff
@@ -1133,9 +1129,16 @@ export const FORCE_ONBOARDING_TO_SHOW = false;
             </div>
 
             <!-- Right side -->
-            <div>
+            <div class="flex flex-row items-center gap-xs">
+              <!-- THESE BUTTONS SHOULD ONLY SHOW TO THE SI TEAM -->
+              <NewButton
+                v-if="featureFlagsStore.ADMIN_PANEL_ACCESS && !DEBUG_MODE"
```

Comment:
I don't know why we would need this...

Reply:
If we need to keep it then we should not use the ADMIN_PANEL_ACCESS flag - it should be a specific flag for this purpose